ticket-booker/src/components/Navbar.style.js

```diff
@@ -0,0 +1,131 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const NavbarContainer = styled.nav`
+  width: 95%;
+  height: 80px;
+  background-color: #222222;
+  height: ${(props) => (props.extendNavbar ? '100vh' : '80px')};
+  display: flex;
+  flex-direction: column;
+  border-bottom: 1px solid #dbdbdb;
+
+  @media (min-width: 556px) {
```

Comment:
556px happens in 3 places. 
Maybe you can create a constant for DRY

Reply:
Created a min width constant which can be reused through the file